assets/quicknavi.js

```diff
@@ -2,6 +2,15 @@ $(function () {
     quicknavi_filter_init();
     $(document).on('pjax:end', function() {
        quicknavi_filter_init();
+       var ctype = getUrlVars()["ctype"];
+       if (ctype)
+       {
+	   $(function() {
```

Comment:
Die CTYPES funktionieren nur in der Struktur, da haben wir ja PJAX - richtig? Außerhalb sollte zumindest der Filter funktionieren. Aber doc-ready ist doch nicht explizit angegeben. Ich initialisiere ja beim Aufruf. Soll ich mergen und du machst einen Gegenvorschlag @staabm  - Bin nicht wirklich ein JS-Spezialist. Die Lösung funktioniert zumindest bei mir.